replset-consistency/validate.js

```diff
@@ -106,6 +106,21 @@ var failArray = [];
 var seen = [];
 var partial = false;
 
+// overloads db.getSiblingDB() which overwrites the runCommand
+// to take readPreference into consideration 
```

Comment:
Can we clarify in this comment that we're implementing this because mongosh v2.0+ no longer allows commands to be run against secondaries?

Reply:
added the info in the next revision